connectors/http/rest/element-templates/http-json-connector.json

```diff
@@ -4,7 +4,7 @@
   "id" : "io.camunda.connectors.HttpJson.v2",
   "description" : "Invoke REST API",
   "documentationRef" : "https://docs.camunda.io/docs/components/connectors/protocol/rest/",
-  "version" : 8,
+  "version" : 10,
   "category" : {
     "id" : "connectors",
```

Comment:
I'm a little concerned that version 10 in this branch doesn't fully match version 10 in `release/8.7` (the difference is that the latter also has the `groupSetCookieHeaders` property).

I don't see how we could do this better as version 9 is also different, so maybe this is just something to keep in mind - version X for one release branch is not guaranteed to be the same as version X in another release branch.